src/python/library/tritonclient/grpc/aio/__init__.py

```diff
@@ -624,7 +624,7 @@ async def infer(
         except grpc.RpcError as rpc_error:
             raise_error_grpc(rpc_error)
 
-    async def stream_infer(
```

Comment:
This used to be an "asynchronous generator function" which the user calls like:
```python
async for response in stream_infer(...):
    ...
```
Now, it is a normal function that returns an "asynchronous iterator", which the user can call it in the exact same way above.